src/routes/conversation/[id]/+page.svelte

```diff
@@ -64,23 +64,39 @@
 		}
 	}
 	// this function is used to send new message to the backends
-	async function writeMessage(message: string, messageId = randomUUID()) {
-		if (!message.trim()) return;
-
+	async function writeMessage({
+		prompt,
+		messageId = randomUUID(),
+		isRetry = false,
+		isContinue = false,
```

Comment:
this is a stylistic comment. In hf/moon-landing, we usually group those kind of arguments into one argument called `opts`. So it would be:
```ts
opts:{
    isRetry?: boolean,
    isContinue?: boolean
}
```

Reply:
Would you also change it at the API level or just for this function ?

Reply:
Let's tackle it in a refacto PR, i think i might need some extra opts for conversation trees anyway :grin: